### PR TITLE
Follow up user improvement

### PR DESCRIPTION
**Pull request by duy-nguyentuong**
- Minor changes for user sign up flow
- Refresh KYCStatus at page open/refresh event
- Refresh token at page open/refresh expired token for API calls

### CODE REVIEW

File: ts/prime/Api/src/services/auth/auth.ts [hunk 1]
```diff
@@ -200,9 +203,16 @@ export class AuthApi {
     if (!authToken) return
     if (authToken !== this.state.authToken) {
       console.log(`got new authToken`)
-      this.setState({ ...this.state, authToken })
+      this.setState({
+        ...this.state,
+        authToken: authToken,
+        expiration: idToken.getExpiration() * 1000,
```
Comment (Git-on-my-level) [inline ts/prime/Api/src/services/auth/auth.ts:209]: Minor but something to think about:
Are we sure about how `getExpiration` works? Does it rely on the local machine's time or is expiration calculated in Cognito? Do we need to worry about timezone or clock drift?

File: ts/prime/ApiReact/src/hooks/forms/signup.ts [hunk 2]
```diff
@@ -67,17 +67,23 @@ export function useSignupFormState(): FormState {
   } = useFormState({
     initialValues,
     handleSubmit: async (values: KeyValues): Promise<any> => {
+      let isValidInput = true
+      const inputErrs: KeyValues = {}
       if (!isValidFirstName(values)) {
-        setErrors({ firstName: 'Invalid first name' })
-        return 'InvalidFirstName'
+        inputErrs.firstName = 'First name must contains alphabetic characters only and starts with capitalized letter'
```
Comment (Git-on-my-level) [inline ts/prime/ApiReact/src/hooks/forms/signup.ts:73]: More specifically due to PrimeTrust's limitations 
"First name must be romanized to contain only Latin characters and start with a capital letter"

File: ts/prime/ApiReact/src/hooks/forms/signup.ts [hunk 3]
```diff
@@ -67,17 +67,23 @@ export function useSignupFormState(): FormState {
...
+        isValidInput = false
       }
       if (!isValidLastName(values)) {
-        setErrors({ lastName: 'Invalid last name' })
-        return 'InvalidLastName'
+        inputErrs.lastName = 'Last name must contains alphabetic characters only and starts with capitalized letter'
```
Comment (Git-on-my-level) [inline ts/prime/ApiReact/src/hooks/forms/signup.ts:77]: "Last name must be romanized to contain only Latin characters and start with a capital letter"

File: ts/prime/Web/src/routes/authed.tsx [hunk 4]
```diff
@@ -26,7 +26,7 @@ import { UserKYCStatuses } from '@/Api'
 import { KeyValues } from '@/Api'
 
 const KYC_MESSAGES: KeyValues = {
```
Comment (Git-on-my-level) [inline ts/prime/Web/src/routes/authed.tsx:28]: Let's try to remove the word "KYC" from the UI, instead we can change these to:
"You haven't submitted your account application. Please submit your account application to continue",
"Your account application was rejected. Please correct your information and resubmit or reach out to our support team",

File: ts/prime/Api/src/services/auth/auth.ts (hunk 1 above)
Comment (duy-nguyentuong) [reply ts/prime/Api/src/services/auth/auth.ts:209]: `getExpiration` will get the expiration from JWT.

**Pull request merged**